api/app/routers/fba.py

```diff
@@ -161,23 +120,22 @@ async def get_hfi_fuels_data_for_fire_centre(run_type: RunType, for_date: date,
         all_zone_data = {}
         for zone_id in zone_ids:
```

Comment:
I'm not sure it matters but we're going to be hitting the database with a query for each zone id. I wonder if there would be any performance gain/loss to query by fire center id and get back all the data in one shot? Maybe not worth the effort.

Reply:
If it's a problem we can address, queries are quite fast the way they are though.